zip-0000.rst

```diff
@@ -623,27 +637,46 @@ Consensus ZIP, a Deployment section MUST be present in order for the ZIP
 to change status to Proposed. Typically, although not necessarily, this
 will specify a network upgrade in which the consensus change is to activate.
 
-A Standards ZIP SHOULD only change status from Proposed to Implemented once
-the Owners provide an associated reference implementation, typically in
-the period after the network upgrade's specification freeze but before
+A ZIP's status is Released if it is Proposed, Active, Implemented, or Final
+(i.e. not Draft, Rejected, Obsolete, or Withdrawn).
+
+A ZIP SHOULD NOT be changed from a non-Released status to a Released
+status if there is significant community opposition to its content.
+(However, Draft ZIPs explicitly MAY describe proposals to which there
+is, or could be expected, significant community opposition.)
+
+A Released ZIP MUST NOT be changed to a non-Released status if the
+specification is already implemented and is in common use, or where a
+Process ZIP still reflects a consensus of the community.
+
+A Standards ZIP SHOULD only change status from Proposed to Implemented
+once the Owners provide an associated reference implementation. For
+Consensus ZIPs, an implementation MUST have been merged into at least
+one consensus node codebase (currently zcashd and/or zebra), typically
+in the period after the network upgrade's specification freeze but before
 the implementation audit. If the Owners miss this deadline, the Editors
 or Owners MAY choose to update the Deployment section of the ZIP to
 target another upgrade, at their discretion.
 
-A Process or Informational ZIP SHOULD change status from Draft to Active
-when it achieves rough consensus on the forum or PR. Such a proposal is
-said to have rough consensus if it has been open to discussion on the
-forum or GitHub PR for at least one month, and no person maintains
-any unaddressed substantiated objections to it. Addressed or obstructive
-objections can be ignored/overruled by general agreement that they have
-been sufficiently addressed, but clear reasoning MUST be given in such
-circumstances.
+A Process or Informational ZIP SHOULD change status from Proposed to
+Active when it achieves rough consensus on the forum or PR. Such a
+proposal is said to have rough consensus if it has been substantially
+complete and open to discussion on the forum or GitHub PR for at least
+one month, has been in Proposed status for at least one week, and no
```

Comment:
4 weeks on the forum and 1 week open as a PR is too short of a time frame to change ZIP status to active, based on the low bandwidth availability of reviewers, the past performance - especially around holiday periods and conference seasons where 2 weeks can pass by without any activity, add the fact of priority of reviewing the status and adoption of existing, active ZIPs.

I suggest a minimum review period of 8 weeks on the forum and 4 weeks as proposed status, unless a change qualifies for an emergency in response to an issue affecting protocol usability by the majority i.e. sand blasting attack and subsequent fee mechanism updates.

This time period would allow a longer time for the community to review the merits and discuss approaches, rather than the need to make multiple new ZIPs as active, possibly even run a Helios poll with the ZCAP or coin weighted polling for consensus and limit hurried shipping of ZIPs, especially as we approach the next block halving period, which is being considered a candidate for a hard fork upgrade.

Reply:
The consensus of the 2023-09-27 ZIP Editors' meeting was to revert this change for now and make it in a separate PR, to allow more discussion on it.